.github/workflows/ci.yml

```diff
@@ -20,8 +20,7 @@ jobs:
           sudo mv nextflow /usr/local/bin/
       - name: Pull docker image
         run: |
-          docker pull nfcore/rnaseq:dev
-          docker tag nfcore/rnaseq:dev nfcore/rnaseq:dev
+          docker pull nfcore/rnaseq:dev && docker tag nfcore/rnaseq:dev nfcore/rnaseq:dev
```

Comment:
```suggestion
          docker pull nfcore/rnaseq:dev
          docker tag nfcore/rnaseq:dev nfcore/rnaseq:dev
```

Reply:
See https://github.com/nf-core/tools/blob/ebf63095ab222e4df987c4e950ce3839e8784d3e/nf_core/pipeline-template/%7B%7Bcookiecutter.name_noslash%7D%7D/.github/workflows/ci.yml#L24-L25

Reply:
Think these tests need to be changed because at the moment they are run for the minimal required NF version and for the latest. Thats probably excessive. Could have a different set of tests for parameters e.g. [here](https://github.com/nf-core/atacseq/blob/59e4fc1a3a10b4d7b2bce59e568e998e2d7cccd5/.github/workflows/ci.yml#L30)